treeherder/config/settings.py

```diff
@@ -37,6 +38,13 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY", default='secret-key-of-at-least-50-characters-to-pass-check-deploy')
 
+# Delete the Pulse automatically when no consumers left
+PULSE_AUTO_DELETE_QUEUES = env.bool("PULSE_AUTO_DELETE_QUEUES", default=False)
+
+# Changing PULSE_AUTO_DELETE_QUEUES to true when treeherder is running in virtual environment
+if hasattr(sys, 'real_prefix'):
```

Comment:
This does not work for me:

```shell
(.venv) armenzg@Armens-MacBook-Pro treeherder % python --version
Python 3.7.6
(.venv) armenzg@Armens-MacBook-Pro treeherder % python -c "import sys; sys.real_prefix"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'sys' has no attribute 'real_prefix'
(.venv) armenzg@Armens-MacBook-Pro treeherder % python -c "import sys; hasattr(sys.real_prefix)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'sys' has no attribute 'real_prefix'
```

This works for me: `os.environ.get("VIRTUAL_ENV")`